projects/Unit/Unit.csproj

```
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
```

You can't execute code that targets `netstandard2.0` so this won't work. This change should be something like:

```suggestion
    <TargetFrameworks>net461;netcoreapp3.1</TargetFrameworks>
```

In theory you could even add `netcoreapp2.1` if you cared about testing more .NET Core versions.

---

Thanks for checking ... I'm just trying to figure out a flaky test. This didn't end up working anyway.

---

It's probably a good idea to run the test suite against `net461` and `netcoreapp3.1` regardless since there could be problems that only occur on one of them.